spec/models/has_two_files_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+describe :has_two_uploadcare_files, :vcr do
```

Comment:
This should also test that post.file1 and post.file2 are different files with urls taken from `attributes[:file1]`/`attributes[:file2]`. Otherwise this specs will pass even without your fix to `uploadcare/rails/active_record/has_file.rb`. 

Same for `has_two_groups_spec.rb`

Reply:
Also validations should be tested, here and for groups